ntoskrnl/include/internal/i386/asmmacro.S

```diff
@@ -249,6 +249,22 @@ MACRO(TRAP_ENTRY, Trap, Flags)
     .ENDP
 ENDM
 
+MACRO(TASK_ENTRY, Trap, Flags)
+    // EXTERN @&Trap&Handler@0 :PROC
+    EXTERN _&Trap&Handler :PROC
+    PUBLIC _&Trap
+    .PROC _&Trap
+        /* Generate proper debugging symbols */
+        FPO 0, 0, 0, 0, 0, FRAME_TSS
+
+        // /* Common code to create the trap frame */
+        // KiEnterTrap Flags
```

Comment:
Here @tkreuzer said to me:
> No trap frame, no handling? I guess since we bugcheck anyway, it's fine, but doesn't this mean we are missing the proper register context?

to which I replied:
> You mean, that eax, ebx and co. are not "correctly initialized" ? Indeed this happens.
>
> This call may be needed if we decide e.g. to make the "TrapFrame" visible in WinDbg.
>
> Actually here the situation would be different I think, and we would have to convert the TSS into a new trap frame. That is what Trap02 does in C (see after "Now build the trap frame based on the original TSS.")